carp.protocols.core/src/commonMain/kotlin/dk/cachet/carp/protocols/domain/sampling/carp/HeartRateSamplingScheme.kt

```diff
@@ -0,0 +1,15 @@
+package dk.cachet.carp.protocols.domain.sampling.carp
+
+import dk.cachet.carp.common.TimeSpan
+import dk.cachet.carp.common.data.HEARTRATE_TYPE
+import dk.cachet.carp.protocols.domain.sampling.DataTypeSamplingScheme
+import dk.cachet.carp.protocols.domain.sampling.IntervalSamplingConfigurationBuilder
+
+class HeartRateSamplingScheme(
```

Comment:
This might be premature. The sampling scheme needs to be derived from some analysis of how various devices can be configured. Can Polar be configured like this? You can go with the `NoOptionsSamplingConfigurationBuilder` for now to leave it open.